db/dataset.py

```diff
@@ -44,7 +44,9 @@ def create_from_dict(dictionary, author_id):
 
         for cls in dictionary["classes"]:
             if "description" not in cls:
-                cls["description"] = None
+                # cls["description"] = None
```

Comment:
We can just delete this, we don't need to keep it. I would also update the check to ensure that someone doesn't intentionally pass in a null value: `if "description" not in cls or cls["description"] is None:`
We should have a test in `db/test/test_dataset` to ensure that, if we pass in a class with no description or a value of None for a description, the dataset is still saved. a variation of `test_create_from_dict` would be good

Reply:
The commented lines have now been deleted. I checked also many parts of various scripts that insert `None` value in the description, and everything has been updated. I hope I don't miss anything. As you guided me, I ran also the tests locally as it is proposed in the README and everything is fine now. 

Yes, a variation of the  `test_create_from_dict` is a good idea. However, you are right about the changes that have to be done in `db/test/test_dataset.py`. When I ran the tests with the updated table schema of the dataset table, I saw a conflict in `_create_dataset()` inside the test_dataset where the argument `desc` was set to `None`. I've changed the argument value to `""` and it worked.

Reply:
right, I think there are 2 tests that we need here:
* If you upload a csv file with no description, the description value should be saved to the database as `""`, this is the change that you fixed
* If someone uses the [dataset create api](https://acousticbrainz.readthedocs.io/api.html#post--api-v1-datasets-) to submit a new dataset and provides no description _or a value of `None` for the description_ then the description should be created with the description changed to `""`. This is where we need a new test, `test_create_from_dict_none_description`, or similar.

Reply:
ok! I'll start checking it later today. Thanks

Reply:
the test case for the above conditions has been created as well as a minor fix in the _db/dataset.py_ that sets the description to `""` when it is inserted as `None` from the _create dataset API_.